src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -37,13 +39,26 @@ def get(self, request, project, version):
         except Release.DoesNotExist:
             raise ResourceDoesNotExist
 
+        ids_from_GroupResolution = GroupResolution.objects.filter(
+            release=release,
+        ).values_list(
+            'group_id', flat=True
+        )
+
+        ids_from_GroupCommitResolution = GroupCommitResolution.objects.filter(
+            commit_id__in=ReleaseCommit.objects.filter(
```

Comment:
We can't make this query without a full sequence scan. There is no index on `commit_id`. We only have a unique on `(group_id, commit_id)`.